browser/ephemeral_storage/ephemeral_storage_tab_helper.cc

```diff
@@ -33,7 +33,8 @@ namespace {
 constexpr char kSessionStorageSuffix[] = "/ephemeral-session-storage";
 constexpr char kLocalStorageSuffix[] = "/ephemeral-local-storage";
 
-const base::TimeDelta kStorageKeepAliveDelay = base::TimeDelta::FromSeconds(30);
+const base::TimeDelta kStorageKeepAliveDelay = base::TimeDelta::FromSeconds(
+    net::features::kBraveEphemeralStorageKeepAliveTimeInSeconds.Get());
```

Comment:
you can make it as a static function or as a class member so the value can be set at runtime and not at process creation.
currently `kStorageKeepAliveDelay ` cannot be controlled via Griffin.